Makefile

```diff
@@ -111,6 +111,7 @@ DEFMEMSZ := 2048
 DEFDISABLEBLOCK := false
 DEFENABLEMEMPREALLOC := false
 DEFENABLESWAP := false
+DEFENABLEDEBUG := false
```

Comment:
My brain initially thought this should be a separate PR, but on reconsideration it's fine - all three commits modify the config - just happens that two of the commits appear related :-)